hw/ip/alert_handler/data/alert_handler.hjson.tpl

```diff
@@ -462,7 +471,10 @@ chars = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H']
                   { value: "0b111", name: "Phase3",   desc: "Escalation Phase3 is active." }
                 ]
         }
-      ]
+      ],
+      tags: [// once an alert or interrupt is triggered, cannot auto-prediect the current
```

Comment:
nit: the current escalation state cannot be auto-predicted

Reply:
that sounds much better thanks michael